services/core/shared/database/knex.js

```diff
@@ -89,7 +89,20 @@ const getDbInstance = async (tableName, migrationDir = './database/knex_migratio
     const find = async (params) => {
         // TODO: Remove after PouchDB specific code is removed from the shared layer
         if (params.selector) params = params.selector;
-        const res = await knex.select().table(tableName).where(params);
+
+        let res;
+        if (params.propBetween) {
+            const { propBetween } = params;
+            delete params.propBetween;
```

Comment:
I think there is no need of `delete params.propBetween;` here. Instead deleting object unnecessarily makes it inefficient.

Reply:
Since `params` is forwarded as the input parameter to the `where()` method in Knex, which translates to the equivalent SQL WHERE clause, not removing it would raise errors because no such column (`propBetween`) exists in the table schema.